perfkitbenchmarker/providers/aws/aws_emr.py

```diff
@@ -84,6 +99,10 @@ def _Delete(self):
     cmd = self.cmd_prefix + ['terminate-clusters', '--cluster-ids',
                              self.cluster_id]
     vm_util.IssueCommand(cmd)
+    if self.bucket_to_delete is not None:
```

Comment:
I don't think you need to explicitly check to make sure that bucket_to_delete is not none - I think you can just do
```
if self.bucket_to_delete:
```